app/src/main/java/com/bnyro/recorder/services/RecorderService.kt

```diff
@@ -91,16 +91,28 @@ abstract class RecorderService : LifecycleService() {
         runCatching {
             unregisterReceiver(bluetoothReceiver)
         }
-        registerReceiver(
-            bluetoothReceiver,
-            IntentFilter(AudioManager.ACTION_SCO_AUDIO_STATE_UPDATED)
-        )
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.UPSIDE_DOWN_CAKE) {
+            registerReceiver(
+                bluetoothReceiver,
+                IntentFilter(AudioManager.ACTION_SCO_AUDIO_STATE_UPDATED),
+                RECEIVER_EXPORTED
+            )
+        } else {
+            registerReceiver(
+                bluetoothReceiver,
+                IntentFilter(AudioManager.ACTION_SCO_AUDIO_STATE_UPDATED)
+            )
+        }
```

Comment:
We can use [ContextCompat#registerReceiver](https://developer.android.com/reference/androidx/core/content/ContextCompat#registerReceiver(android.content.Context,android.content.BroadcastReceiver,android.content.IntentFilter,int) instead).